packages/api/src/webhooks/cannon.ts

```diff
@@ -23,30 +24,37 @@ export default class WebhookCannon {
   running: boolean;
   verifyUrls: boolean;
   resolver: any;
-  constructor({ db, store, verifyUrls }) {
+  queue: MessageQueue;
+  constructor({ db, store, verifyUrls, queue }) {
     this.db = db;
     this.store = store;
     this.running = true;
     this.verifyUrls = verifyUrls;
     this.resolver = new Resolver();
+    this.queue = queue;
     // this.start();
   }
 
   async start() {
     console.log("WEBHOOK CANNON STARTED");
-    if (this.running) {
-      await this.db.queue.setMsgHandler(this.processEvent.bind(this));
-    }
+    await this.queue.consume(this.handleQueueMsg.bind(this));
```

Comment:
Does this setup the consumption for all events from now on or it only consumes one? I thought it was the latter until now but if the first instead WDYT of renaming it to something like `setupConsume`, `startConsuming` or something like that?

Reply:
it's for all the events from now (or if there are unconsumed msgs on the queue).
it's named `consume` following the `consume` function on the `amqplib` we use. i'm cool with changing it if the semantics makes a difference here though.